config/authorities.yml

```diff
@@ -0,0 +1 @@
+:local_path: "config/authorities"
```

Comment:
@coblej Asked about this part and what it does. After some investigation, I've confirmed it's not needed so I'll remove this part and resubmit the PR. This gets written by the generator in the [documentation for YAML-driven local subauthorities](https://github.com/samvera/questioning_authority#in-yaml-files). According to the documentation:
> By default, local authority YAML files are located in config/authorities/

...and since we're not deviating from the default, we don't actually need to specify it here.